tests/lib/tools/tests.backup

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -e
+
+show_help() {
+    echo "usage: tests.backup prepare"
+    echo "       tests.backup restore"
+}
+
+cmd_prepare() {
+    local BACKUP_PATH=${1:-$(pwd)}
+
+    if [ ! -d "$BACKUP_PATH" ]; then
+        echo "tests.backup: cannot prepare, backup dir does not exist" >&2
+        exit 1
+    fi
+    tar cf "${BACKUP_PATH}.tar" "$BACKUP_PATH"
+}
+
+cmd_restore() {
+    local BACKUP_PATH=${1:-$(pwd)}
+    if [ -f "${BACKUP_PATH}.tar" ]; then
+        find "${BACKUP_PATH}" -maxdepth 1 -mindepth 1 -exec rm -rf {} \;
+        tar -C/ -xf "${BACKUP_PATH}.tar"
+        rm "${BACKUP_PATH}.tar"
+
+        # $BACKUP_PATH was cleaned and recreated, enter the directory again
+        cd "$BACKUP_PATH"
```

Comment:
This is pointless as current working directory is only meaningful to a process and we are no longer a source tool.

Reply:
agree, thanks for the suggestion